tests/conformance/conformance.go

```diff
@@ -20,7 +20,7 @@ func done(files []string) {
 func main() {
 	err := execute()
 	done([]string{
-		"/tmp/results/junit.xml",
+		"/tmp/sonobuoy/results",
```

Comment:
Do we still need this? The done function is already aware of the directory, so this does not add any extra value?

Reply:
Here you say what you want to collect
This is two different things, that just equal in our specific case
* resultsDir - which is by default /tmp/sonobuoy/results, there you write the done file (this what `done` function is doing)
* the done content is what you want to collect, we used to collect just a file, but since v0.56.9 if you write a folder it will auto tar it and collect it all
*

Reply:
Understood, thanks